src/calculateRentalCost.js

```diff
@@ -29,6 +32,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const threeDay = 3;
+  const sevenDay = 7;
```

Comment:
```suggestion
  const longTerm = 7;
```
А якщо період для великої знижки зміниться, наприклад, на 10 днів? Тоді така назва не підійде